### PR TITLE
Adds autorefetching for dynamo zero-result-filter.

### PR DESCRIPTION
**Pull request by pckilgore**
After running for nearly a month locally against the entire codebase without errors this should be safe to merge, and it will help bugfix a bunch of stuff.

@ikeforrest FYI if we need to revert back to 1.24.x

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@groundbreaker/qewl",
-  "version": "1.24.1",
+  "version": "1.25.1",
```
Comment (ikeforrest) [inline package.json]:
```suggestion
  "version": "1.25.0",
```

File: src/query/index.js [hunk 2]
```diff
@@ -75,6 +80,7 @@ const decorateDetailBase = ({
   );
 };
 
+let pendingAutoFetch = false;
```
Comment (ikeforrest) [inline src/query/index.js]: why does this need to be set in the global scope?

File: src/query/index.js [hunk 3]
```diff
@@ -97,12 +103,44 @@ const decorateListBase = ({
...
+          nextToken
+        ) {
+          pendingAutoFetch = true;
+          fetchMore({
+            variables: { ...params, nextToken },
+            updateQuery: (previousResult, { fetchMoreResult }) => {
+              const next = fetchMoreResult[query];
+              const previous = previousResult[query];
+
+              console.log("Merging", {
```
Comment (ikeforrest) [inline src/query/index.js]: 👋

File: src/query/index.js (hunk 2 above)
Comment (pckilgore) [reply src/query/index.js]: I tried to get this working a couple of other ways that didn't work and this did.  It's obviously not a long term solution but right now it's a blocker, and I feel like the long term solution requires an moving infinite scroll triggers to the application and out of qewl.

Given this solves the problem app-wide and hasn't broken on me yet once in 27 days, and will probably be removed as a feature entirely, eventually, I'm OK with the technical debt.

**Pull request merged**